app/src/components/modals/TransferFromBankModal.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+  <dialog id="my_modal_4" class="modal modal-bottom sm:modal-middle modal-open">
+    <div class="modal-box">
+      <button
+        class="btn btn-sm btn-circle btn-ghost absolute right-2 top-2"
+        @click="$emit('closeModal')"
+      >
+        ✕
+      </button>
+
+      <h1 class="font-bold text-2xl">Transfer from Bank Contract</h1>
+      <h3 class="pt-8">
+        This will transfer {{ amount }} {{ NETWORK.currencySymbol }} from the team bank contract to
+        this address {{ to }}.
+      </h3>
+
+      <label class="input input-bordered flex items-center gap-2 input-md mt-2">
+        <p>To</p>
+        |
+        <input type="text" class="grow" v-model="to" />
+      </label>
+
```

Comment:
I think u can display the bank contract's balance in this modal.

Reply:
is this alright?
<img width="525" alt="image" src="https://github.com/globe-and-citizen/cnc-portal/assets/49224826/b925f5ed-ec4e-41a8-b5e8-8a2250e64743">